src/suites/cts/validation/create_bind_group_layout.spec.ts

```diff
@@ -0,0 +1,87 @@
+export const description = `
+Create bind group layout validation tests.
```

Comment:
I'd use "createBindGroupLayout" both here and in the filename, because that's the name of the function being tested.

Reply:
Done.